src/components/ui/Text.tsx

```diff
@@ -413,12 +415,22 @@ export function TinyText({
   return <div className={tinyText(color, fontPrimary)}>{children}</div>
 }
 
-const textButton = classnames(
-  textDecoration('underline'),
-  opacity('disabled:opacity-75')
-)
-export function TextButton(props: React.HTMLAttributes<HTMLButtonElement>) {
-  return <button className={textButton} {...props} />
+const textButton = (small?: boolean, center?: boolean) =>
+  classnames(
+    textAlign({ 'text-center': center }),
+    fontSize({ 'text-sm': small }),
+    textColor('text-tertiary', 'hover:text-accent'),
+    textDecoration('underline'),
+    opacity('disabled:opacity-75'),
+    cursor('cursor-pointer')
+  )
+export function TextButton(
+  props: React.HTMLAttributes<HTMLSpanElement> & {
```

Comment:
Can't you just use `HTMLAttributes<HTMLSpanElement>` by importing it instead of `React`?